src/service/resources-impl.js

```diff
@@ -1602,6 +1642,11 @@ export class ResourcesImpl {
    * @private
    */
   taskComplete_(task, success, opt_reason) {
+    this.totalLayoutCount_++;
+    if (task.resource.isInViewport()) {
+      this.slowElementCount_++;
```

Comment:
This will penalize elements that are in the first viewport when the page isn't actually visible. Perhaps we should be checking if the viewer has ever been visible, too.

Reply:
Interesting point! I'm not sure if that level of accuracy buys us anything considering all non-prerendered pages will still have a similar issue:

> Slow element ratio is hugely affected by the percentage of total elements that are within the initial viewport. We could attempt to normalize this by removing these elements from the stat

Reply:
No, I think we need to count these elements as eager if the viewer isn't visible.

Reply:
It will be interesting to see the difference between prerendered and non-prerendered pages.

Reply:
@jridgewell, does this mean that you think for a page that hasn't been visible yet:
- ser =0
- All laid out elements were eager (eer probably is 1)

Reply:
resolving since we talked about this offline.